### DISCUSSION TITLE
Scheduled Recording

### DISCUSSION THREAD

**Discussion by ibn-e-battuta** [bluenviron/mediamtx#3973] [Questions]
### Question

Is it possible to configure the recording functionality to start and stop at specific times? For example, I would like the system to automatically record only between 6 AM and 6 PM, avoiding any recording outside this window.

**Comment by dbozz** [comment]
I'd like to know this too. But until then I am using an external cron script that runs a script with ffmpeg. It works fine, even if an in house solution would be better.
I guess one could also try modifying the script with hot reload and turn on/off recording on the fly... not the prettiest solution tho...

**Comment by aler9** [comment]
moved in #4599
